src/modules/commander/commander.cpp

```diff
@@ -1731,6 +1733,9 @@ int commander_thread_main(int argc, char *argv[])
 									(hrt_abstime)disarm_when_landed * 1000000);
 			}
 
+			int32_t v;
+			param_get(_param_acro_att, &v);
+			acro_att = (v != 0);
```

Comment:
@anton-matosov That's not entirely true. The param metadata supports @boolean which should be used for anything that has only two values (on and off). Your proposal should only be used for something that is not on and off, but a selection between two different things.